_docs/tasks/configuring-mixer.md

```diff
@@ -0,0 +1,167 @@
+---
```

Comment:
I tend to agree. This feels more like an explanation than a Task.  The information is valuable though... maybe as a preface for a reference doc on Config?